framework/src/wifi_manager/wifi_manager_state.c

```diff
@@ -320,6 +321,55 @@ wifi_manager_result_e _wifimgr_scan(wifi_manager_scan_config_s *config)
 	return WIFI_MANAGER_SUCCESS;
 }
 
+static wifi_manager_result_e _wifimgr_scan_multi_aps(wifi_manager_scan_multi_configs_s *configs)
+{
+	if (!configs) {
+		WIFIMGR_CHECK_UTILRESULT(wifi_utils_scan_multi_aps(NULL), TAG,
+								 "request scan multi aps to wifi utils is fail");
+		return WIFI_MANAGER_SUCCESS;
+	}
+
+	trwifi_scan_multi_configs_s uconf = {0};
+	memset(&uconf, 0, sizeof(trwifi_scan_multi_configs_s));
+	for (int i = 0; i < configs->scan_ap_config_count; i++) {
+		wifi_manager_scan_config_s *config = &configs->ap_configs[i];
+		if (config->ssid_length > 0) {
```

Comment:
I see that ssid_length was checked in wifi_manager_scan_multi_aps(), do we need to check it again here?

Reply:
Here, the purpose is to determine whether or not to copy the SSID string.